base/deprecated.jl

```diff
@@ -259,3 +257,9 @@ const base64 = base64encode
 
 @deprecate sizehint(A, n) sizehint!(A, n)
 
+@deprecate randbool!             rand!
+@deprecate randbool()            rand(Bool)
+@deprecate randbool(dims)        bitrand(dims)
+@deprecate randbool(dims...)     bitrand(dims)
+@deprecate randbool(r, dims)     bitrand(r, dims)
+@deprecate randbool(r, dims...)  bitrand(r, dims)
```

Comment:
Never mind. We do have it - but I was trying in 0.3.4, which was open in another window.